data/transit/route/tram.json

```diff
@@ -36,18 +36,6 @@
         "route": "tram"
       }
     },
-    {
-      "displayName": "Bernmobil",
```

Comment:
I search taginfo and found 1227 elements still have a `operator=Bernmobil`, so if you just remove it, in next weekly collection run, program will pick it out from planet.osm file again. 
Is this operator merged or bought by another operator? Or just changed to a new name?

Reply:
Bernmobil is not an operator, it's a brand name. The operator is still SVB, which unfortunately got changed by iD users based on the suggestion. Also the lines are part of the 'Libero' network, but not all Libero lines are operated by SVB/Bernmobil.